src/vscode-dts/vscode.proposed.chatAgents2Additions.d.ts

```diff
@@ -10,8 +10,16 @@ declare module 'vscode' {
 		readonly action: InteractiveSessionCopyAction | InteractiveSessionInsertAction | InteractiveSessionTerminalAction | InteractiveSessionCommandAction | InteractiveSessionFollowupAction;
 	}
 
+	export interface ChatAgentResult2Feedback {
+		/**
+		 * Whether the user wants to report an issue.
+		 */
+		readonly reportIssue?: boolean;
```

Comment:
Similar to the inline API we should have another type for `ChatAgentUserActionEvent#action` instead of a sibling property. The new action would then be exposed via `ChatAgent2#onDidPerformAction` (which is the additions API)